lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts

```diff
@@ -285,7 +294,7 @@ export class DropdownCloudWidgetComponent extends WidgetComponent implements OnI
     }
 
     private resetValue() {
-        this.field.value = '';
+        this.field.value = null;
```

Comment:
this sounds very scary. until now we were resetting the field value with an empty string ( so it will never be null ) and now u are passing null, why?

Reply:
I thought that since value is indeed of type **FormFieldOption** (not id - string) I can switch to null